assets/css/viewer.css

```diff
@@ -1,23 +1,35 @@
+.viewer--main {
+  background-color: white;
+  padding-top: 1em;
+}
+
+.viewer--line-switcher {
+  margin-bottom: 2em;
+}
+
 .viewer--sign {
-  border: 1px solid gray;
-  float: left;
-  margin: 1em 1em 0 0;
-  width: 15em;
+  background-color: rgb(206, 206, 206);
+  margin-bottom: 1em;
+  width: 20em;
 }
 
 .viewer--sign-id {
-  font-weight: bold;
-  padding: 0.5em;
+  border-top: 10px solid orange;
```

Comment:
just noticed that this is `orange` wheras other colors are given rgb values.  is orange what we wanted?

Reply:
if you want to be really fancy the colors are available via the API as in `https://api-v3.mbta.com/routes/Orange`

Reply:
but that's probably overkill - you could just take it from that API call which has it as `ED8B00`

Reply:
The actual colors get set with JS, and can be found in `colors.js`. I got their values from the [MBTA color style guide](https://projects.invisionapp.com/dsm/mbta-customer-technology/digital-style-guide/folder/colors/5ac782001c780e0011d2f352), and I checked the orange against the one in the API here and it was the same, so I assume the rest are as well.

I think a border requires a width, style, and color, so I just used a default color here in order to set the width and style, so that the JS can later change it to route specific values. The color here should never actually display.

I'll just change this to `white`, then.